docs/cookbook.rst

```diff
@@ -249,3 +249,55 @@ Unless of course something bad happens inside of ``PerformA`` in which case the
     AssertionError:
     Expected: <True>
          but: was <False>
+
+
+Using Either
+===========
+
+Sometimes you may need to use a try/except control flow in your test, for 
+one reason or another. Luckily, your Actor can perform this flow 
+with the :class:`~screenpy.actions.Either` Action!::
```

Comment:
This is a small nitpick again, but the `::` in RST does turn into a `:`, so this line will look like this:
> ...your Actor can perform this flow with the Either Action!:

Reply:
I'm not sure how to make the following code block without the use of the double colon.  

```
with the :class:`~screenpy.actions.Either` Action!::

    the_actor.will(Either(DoAction()).or_(DoDifferentAction())
```

Reply:
Had to read up on RST.  Fixing.

Reply:
I updated all the code blocks to keep everything consistent.